docs/blog/articles/2023-02-05-KServe-0.10-release.md

```diff
@@ -0,0 +1,115 @@
+        input_tensors = [image_transform(instance) for instance in request.inputs[0].data]
+        input_tensors = np.asarray(input_tensors)
+        infer_inputs = [InferInput(name="INPUT__0", datatype='FP32', shape=list(input_tensors.shape),
+                                   data=input_tensors)]
+        infer_request = InferRequest(model_name=self.model_name, infer_inputs=infer_inputs)
+        return infer_request
+```
+
+You can use the same Python API type `InferRequest` and `InferResponse` for both REST and gRPC protocol, KServe handles the underlying decoding and encoding according to the protocol.
+New `headers` argument is added to the custom handlers to pass http/gRPC headers or metadata, you can also use this as context dict to pass data between handlers.
+
+
+Please check the following table for complete support matrix for ServingRuntimes and ModelFormats.
+
+| Model Format        | v1           | v2 REST/gRPC | 
+| ------------------- |--------------| ----------------|
+| Tensorflow          | ✅ TFServing    | ✅ Triton |
+| PyTorch             | ✅ TorchServe   | ✅ TorchServe |
+| TorchScript         | ✅ TorchServe   | ✅ Triton |
+| ONNX                | ❌              | ✅ Triton |
+| Scikit-learn        | ✅ KServe       | ✅ MLServer |
+| XGBoost             | ✅ KServe       | ✅ MLServer |
+| LightGBM            | ✅ KServe       | ✅ MLServer |
+| MLFLow              | ❌              | ✅ MLServer |
+| Custom              | ✅ KServe       | ✅ KServe |
+
+
+## Multi-Arch Image Support
+
+KServe control plane images [kserve-controller](https://hub.docker.com/r/kserve/kserve-controller/tags),
+[kserve/agent](https://hub.docker.com/r/kserve/agent/tags), [kserve/router](https://hub.docker.com/r/kserve/router/tags) are now supported 
+for multiple architectures: `ppc64le`, `arm64`, `amd64`, `s390x`.
+
+## KServe Storage Credentials
+
+- Currently, AWS users need to create a secret with long term/static IAM credentials for downloading models stored in S3.
+  Security best practice is to use [IAM role for service account(IRSA)](https://aws.amazon.com/blogs/opensource/introducing-fine-grained-iam-roles-service-accounts/) 
+  which enables automatic credential rotation and fine-grained access control, see how to [setup IRSA](https://kserve.github.io/website/0.10/modelserving/storage/s3/s3/#create-service-account-with-iam-role).
+- Support Azure Blobs with [managed identity](https://docs.microsoft.com/en-us/azure/active-directory/managed-identities-azure-resources/how-manage-user-assigned-managed-identities?pivots=identity-mi-methods-azcli).
+
+## ModelMesh updates
+
+- Support for TorchServe ServingRuntime.
+- Support v2 inference protocol for OpenVINO ServingRuntime.
+- `ClusterServingRuntime` support for ModelMesh which now works in the same way as KServe.
+- Support passing labels and annotations from `ServingRuntimePodSpec` to ModelMesh Pods.
+- Support for `ImagePullSecrets` on ServingRuntime spec.
```

Comment:
ModelMesh has continued to integrate itself as KServe's multi-model serving backend, introducing improvements and features that better align the two projects. For example, it now supports ClusterServingRuntimes, allowing use of cluster-scoped ServingRuntimes, originally introduced in KServe v0.8. 

Additionally, ModelMesh has introduced support for TorchServe enabling users to serve arbitrary PyTorch models (e.g. eager-mode) in the context of distributed-multi-model serving. 

Other limitations have been addressed as well, such as adding support for BYTES/string type tensors when using the REST inference API for inference requests that require them.